src/app/app.component.spec.ts

```diff
@@ -5,14 +5,14 @@ import { AppModule } from './app.module';
 
 describe('AppComponent', () => {
     beforeEach(async(() => {
-        TestBed.configureTestingModule({
+        void TestBed.configureTestingModule({
```

Comment:
This is new. I haven't seen us defining types on function calls before. Is this because of the recent changes in typescript/eslint rules?

Reply:
This is related to the rule changes in the latest version of the eslint-parser, ^3.0.0.  This and a handful of other rules will need to be tweaked in our own eslint config.

Reply:
Should we update the rules quickly so we don't have to add and then remove all of these extra types in the pattern examples?

Reply:
I'll make a quick PR in the code-standards library to account for the new rules.  I'm still approving this PR but yeah we should get these changes in before any new design patterns are made.